src/pages/Explorer/Components/TypeSelector.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import Svg from 'webkit/ui/Svg/svelte'
+  import Tootlip from 'webkit/ui/Tooltip/svelte'
+  import Checkbox from 'webkit/ui/Checkbox.svelte'
+  import { EntityType } from '../const'
+
+  export let onChange = () => {}
+  export let types = new Set(Object.values(EntityType))
+
+  const toggleType = (type) => {
+    if (types.has(type)) {
+      types.delete(type)
+    } else {
+      types.add(type)
+    }
+    types = types
+    onChange(types)
+  }
+</script>
+
+<Tootlip on="click" activeClass="$style.active" align="end">
+  <div slot="trigger" class="btn-2 btn--s">Types: {types.size}</div>
+
+  <div slot="tooltip" class="tooltip">
+    <div class="caption txt-m mrg-s mrg--l mrg--b c-waterloo">Types</div>
+    {#each Object.values(EntityType) as type, index (index)}
```

Comment:
Never use `index` as a key. If the items order will not change, keying is not needed.
```suggestion
    {#each Object.values(EntityType) as type}
```